.ci/buildAll.sh

```diff
@@ -3,7 +3,7 @@
 # Find paths that contain the Gradle wrapper executable.
 APP_PROJECTS=$(find $PWD -maxdepth 1 -type d -exec [ -x {}/gradlew ] \; -print -prune)
 
-pushd . > /dev/null
```

Comment:
Ah, right. So another fix would have been to change the shebang line to `#!/bin/bash`, but I'm fine with your solution.